ranger/gui/widgets/pager.py

```diff
@@ -246,9 +246,10 @@ def _generate_lines(self, starty, startx):
         while True:
             try:
                 line = self._get_line(i).expandtabs(4)
+                line_len = ansi.char_len(line) if self.markup == 'ansi' else len(line)
```

Comment:
Doesn't `ansi.char_len` always give the right answer? I don't think we need the fallback to `len`?

Reply:
It probably gives the right answer, but I simply followed the pattern from a few lines below, where the char slice is taken :
https://github.com/ranger/ranger/blob/1654128f955b320354eea98a0bb6970a159ab16f/ranger/gui/widgets/pager.py#L253-L259


I assumed it was done for performance reasons, but I'm not sure.

Would you like me to always use `ansi.char_len()`?

Reply:
I took that as a yes :)
Done.

Reply:
@toonn Is there anything I can do to help get this PR merged?

Reply:
Only waiting for me (or someone else on the team *gasp*) to get a round Tuit : /